### DISCUSSION TITLE
Fladder documentation?

### DISCUSSION THREAD

**Discussion by bazzadazza72** [DonutWare/Fladder#22] [Ideas]
Hi there,

I was thinking about writing a wiki for users that want to quickly get started with Fladder.

My goal for this is to provide instructions for people that are familiar with the first-party Jellyfin apps and want a better experience, and also for the users that are not that familiar with Jellyfin and want something quick to learn.

Let me know your thoughts on whether we should implement this.

**Comment by PartyDonut** [comment]
Yes, good idea. That would be helpful. 
Something like installation instructions for platforms that require it.
